runtime-kernel/linux-kernel-rpi64-lts/autobuild/build

```diff
@@ -0,0 +1,111 @@
+#
+# The Raspberry Pi build script
+# Copied from `linux-kernel' build.
+#
+abinfo "Preparing build environment ..."
+OLDARCH="$ARCH"
+unset ARCH
+if [[ "${CROSS:-$ARCH}" = "amd64" ]]; then
+    SRCARCH="x86"
+elif [[ "${CROSS:-$ARCH}" = "arm64" ]]; then
+    SRCARCH="arm64"
+elif [[ "${CROSS:-$ARCH}" = "loongson3" ]]; then
+    SRCARCH="mips"
+elif [[ "${CROSS:-$ARCH}" = "ppc64el" ]]; then
+    SRCARCH="powerpc"    
+fi
+
+# RPi4 kernel build starts.
+
+abinfo "Building kernel for Raspberry Pi 4."
+
+abinfo "Copying config and setting up env..."
+
+cp -v autobuild/config-rpi4 "$SRCDIR"/.config
+
+. autobuild/build-common/set-var
+
+abinfo "Building kernel..."
+make
+
+abinfo "Stripping symbols..."
+. autobuild/build-common/strip-modules
+
+abinfo "Installing modules..."
+mkdir -p "$PKGDIR"/usr
+make INSTALL_MOD_PATH="$PKGDIR"/usr modules_install
+
+abinfo "Copying kernel..."
+# make a local copy of compiled kernel
+mkdir -p "$PKGDIR/usr/lib/aosc-os-arm64-boot/linux-kernel-$version$LOCALNAME/"
```

Comment:
You can remove this line and the line below installing the kernel to this directory. It is not being used, since it was an old standard.